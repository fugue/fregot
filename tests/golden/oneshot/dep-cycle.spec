{
    "command": "fregot",
    "arguments": [
        "test",
        "dep-cycle-01.rego",
        "dep-cycle-02.rego",
        "dep-cycle-03.rego"
    ],
    "asserts": [
        {
            "exit_code": 1
        },
        {
            "stderr": "${SPEC_NAME}.stderr"
        }
    ]
}
