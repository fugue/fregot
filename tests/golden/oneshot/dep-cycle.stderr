fregot ([34minterpreter error[0m):
  Cyclic dependency error:
    fregot.tests.invalid.dep_cycle_01 depends on
    fregot.tests.invalid.dep_cycle_02 depends on
    fregot.tests.invalid.dep_cycle_03 depends on
    fregot.tests.invalid.dep_cycle_01

fregot ([34minterpreter error[0m):
  Cyclic dependency error:
    fregot.tests.invalid.dep_cycle_02 depends on
    fregot.tests.invalid.dep_cycle_03 depends on
    fregot.tests.invalid.dep_cycle_01 depends on
    fregot.tests.invalid.dep_cycle_02

fregot ([34minterpreter error[0m):
  Cyclic dependency error:
    fregot.tests.invalid.dep_cycle_03 depends on
    fregot.tests.invalid.dep_cycle_01 depends on
    fregot.tests.invalid.dep_cycle_02 depends on
    fregot.tests.invalid.dep_cycle_03
